tests/queries/0_stateless/03403_function_tokenizeChinese.sql

```diff
@@ -0,0 +1,47 @@
+SELECT 'With constant values.';
+SELECT tokenizeChinese('他来到了网易杭研大厦');
```

Comment:
Let's rewrite to this form to avoid some copypasta.

```sql
WITH `他来到了网易杭研大厦` AS text
SELECT tokenizeChinese(text) AS tokenized, length(tokenized) as length;
```